### ISSUE TITLE
Missing page: Accessibility

### ISSUE THREAD

**Issue by mivoligo** [Feichtmeier/settings#35]
I can give it a try in the near future. 

This is how it looks in gnome control center. It's named "Universal Access" for me though.
![Settings_004](https://user-images.githubusercontent.com/1629120/131118680-fcc67caf-1f7a-4c6f-88d8-de244bf01acd.png)

**Comment by mivoligo** [comment]
This is how it looks so far. More details in PR #36.

![Screenshot from 2021-09-07 18-15-28](https://user-images.githubusercontent.com/1629120/132385091-eaf310d6-e179-47e7-92dc-36ada5d09a5c.png)

**Comment by Jupi007** [comment]
Very impressive!

**Comment by Feichtmeier** [comment]
Really nice! Tryig this in the next days!

**Comment by Feichtmeier** [comment]
@mivoligo finally found a little time window to open ubuntu - works really nice, impressive!
![Bildschirmfoto von 2021-09-10 10-24-13](https://user-images.githubusercontent.com/15329494/132823989-cb9ca27e-81a4-4f1c-a643-35b62efcb2c3.png)
Thank you very much!
My contributions will probably start in october again :monocle_face: :brain:  :laughing:
